### ISSUE TITLE
У больничных ламп складывается эффективность

### ISSUE THREAD

**Issue by A1EE** [skyarkhangel/Hardcore-SK#1828]
По задумке есть 3 больничные ламы(Лампа лекаря, хирургическая лампа, плазменный шар), которые взаимозаменяются на более эффективные при открытии новых технологий(как это работает с инструментами). Однако на деле замены не происходит, и можно присоединить к больничной кровати аж 3 лампы разных эпох, и получить с каждой бонус "прироста качества лечения".

**Comment by Skippeer12** [comment]
Инструменты тоже складываются 👀

**Comment by A1EE** [comment]
> Инструменты тоже складываются eyes
> Можно конечно, но тогда у тебя получится что больше места занимают инструменты, а не больничные кровати или госпиталь прийдеться делать большим. А место как мы знаем, очень важно в римаче

Нет, инструменты как раз не складываются. Подключаются, но бонус идет только от самых продвинутых. А у ламп от всех 3х.

**Comment by Skippeer12** [comment]
>Нет, инструменты как раз не складываются. Подключаются, но бонус идет только от самых продвинутых. А у ламп от всех 3х.

![image](https://user-images.githubusercontent.com/18488106/60967542-9b86db00-a323-11e9-9e0e-d5284bae7f94.png)

Инструменты тоже стакаются, только у них там немного другие параметры идут, а у плазменных инструментов есть те же параметры как и у ламп, по этому отображает только их.
![image](https://user-images.githubusercontent.com/18488106/60967780-236ce500-a324-11e9-8dff-31d7490889cd.png)
Вот стакается фактор иммунитета

**Comment by Pepelnica1488** [comment]
не стану создавать новую тему но не логично было бы добавить new colony hope бафф для новых основанных колоний помимо стартовых? мне вот пришлось убегать от осаждающих потому что у них были снайперки и калаши с гранатометами а у меня 9мм пукалки,я пришел на новую локацию роняя кал но мои колонисты остались капризными мразями не понимающими что их только что жестоко опустили

**Comment by Zim777** [comment]
> не стану создавать новую тему но не логично было бы добавить new colony hope бафф для новых основанных колоний помимо стартовых? мне вот пришлось убегать от осаждающих потому что у них были снайперки и калаши с гранатометами а у меня 9мм пукалки,я пришел на новую локацию роняя кал но мои колонисты остались капризными мразями не понимающими что их только что жестоко опустили и надо бы не выпендриваться лишний раз

Я думаю, надо бы вешать дебафф в -20 "мы убегали из нашего дома"

**Comment by Pepelnica1488** [comment]
> > не стану создавать новую тему но не логично было бы добавить new colony hope бафф для новых основанных колоний помимо стартовых? мне вот пришлось убегать от осаждающих потому что у них были снайперки и калаши с гранатометами а у меня 9мм пукалки,я пришел на новую локацию роняя кал но мои колонисты остались капризными мразями не понимающими что их только что жестоко опустили и надо бы не выпендриваться лишний раз
> 
> Я думаю, надо бы вешать дебафф в -20 "мы убегали из нашего дома"

бред

**Comment by Zim777** [comment]
Увы, но это логика, твои колонисты убегали, потому что их опустили,  Они должны быть деморализованы. Или они должны прыгать и радоваться - "нас опустили, поэтому нам хорошо" +20?

**Comment by Pepelnica1488** [comment]
> Увы, но это логика, твои колонисты убегали, потому что их опустили, Они должны быть деморализованы. Или они должны прыгать и радоваться - "нас опустили, поэтому нам хорошо" +20?

ты хоть раз начинал за племя?

**Comment by Zim777** [comment]
К вашему сожалению, Пекло-Ренди(Рей)-племя-только кухляночки - для меня вполне стандартный легкий старт. А то, что вы не способны нормально и быстро обустраивать быт колонистов - конкретно ваши проблемы, а не всех остальных.

**Comment by hadaev8** [comment]
Инструменты стакаются тоже.
https://i.imgur.com/vxWredJ.png

**Comment by Pepelnica1488** [comment]
> К вашему сожалению, Пекло-Ренди(Рей)-племя-только кухляночки - для меня вполне стандартный легкий старт. А то, что вы не способны нормально и быстро обустраивать быт колонистов - конкретно ваши проблемы, а не всех остальных.

прочитай что написано при старте за племя и не неси больше свою чушь

**Comment by Zim777** [comment]
> > К вашему сожалению, Пекло-Ренди(Рей)-племя-только кухляночки - для меня вполне стандартный легкий старт. А то, что вы не способны нормально и быстро обустраивать быт колонистов - конкретно ваши проблемы, а не всех остальных.
> 
> прочитай что написано при старте за племя и не неси больше свою чушь

Извините, у вас все в порядке с логикой? Вы хотите положительный баф "новая колония", после того, как покинули свое поселение, убегая от врага?
И к чему тогда вдруг старт за за племя к выше указанной ситуации?
Убедительная просьба, сначала научитесь логически мыслить, не терять нить разговора, не прыгать с одной темы на другую, и только тогда вступайте в беседу с людьми, а главное не опускаться до оскорблений, особенно когда вы по факту не правы.

**Comment by Pepelnica1488** [comment]
> > > К вашему сожалению, Пекло-Ренди(Рей)-племя-только кухляночки - для меня вполне стандартный легкий старт. А то, что вы не способны нормально и быстро обустраивать быт колонистов - конкретно ваши проблемы, а не всех остальных.
> > 
> > 
> > прочитай что написано при старте за племя и не неси больше свою чушь
> 
> Извините, у вас все в порядке с логикой? Вы хотите положительный баф "новая колония", после того, как покинули свое поселение, убегая от врага?
> И к чему тогда вдруг старт за за племя к выше указанной ситуации? Вы не способны выжить за племя без указанного бафа?
> Убедительная просьба, сначала научитесь логически мыслить, не терять нить разговора, не прыгать с одной темы на другую, и только тогда вступайте в беседу с людьми, а главное не опускаться до оскорблений, особенно когда вы по факту не правы.

мб это у тебя не в порядке что то с логикой? ибо ЭТО УЖЕ ЕСТЬ В ИГРЕ И ИМЕННО УБЕГАЯ ОТ ВРАГА ОНИ ПОЛУЧАЮТ БАФФ ОСНОВЫВАЯ НОВУЮ КОЛОНИЮ ты БРЕДИШЬ ИЛИ ПРОСТО ДЕГЕНЕРАТ?

**Comment by Zim777** [comment]
>не стану создавать новую тему но не логично было бы добавить new colony hope бафф для новых основанных колоний помимо стартовых?" (С) Pepelnica1488
> ЭТО УЖЕ ЕСТЬ В ИГРЕ И ИМЕННО УБЕГАЯ ОТ ВРАГА ОНИ ПОЛУЧАЮТ БАФФ ОСНОВЫВАЯ НОВУЮ КОЛОНИЮ (с) Pepelnica1488

Так ты "БРЕДИШЬ ИЛИ ПРОСТО ДЕГЕНЕРАТ?"

**Comment by ak72ti** [comment]
в ссоре дегенераты оба. (с) (никого не обижаю просто цитатка ага)
....
а по делу скажу так, психологически бафф сходный с новой колонией все же имеет место быть, все таки эти люди выжили после всего. Но это уже из ряда сотен идей которые никогда не реализуют.
